sematic/ui/src/components/LogPanel.tsx

```diff
@@ -1,51 +1,24 @@
-import { Alert, Box } from "@mui/material";
-import { useCallback, useContext, useState } from "react";
-import { UserContext } from "..";
+import { Alert, Box, TextField } from "@mui/material";
```

Comment:
@chance-an FYI on `main`, the error message is rendered without proper spacing, compared to info messages:

![image](https://user-images.githubusercontent.com/1894533/207062810-141a26b3-ba44-4e78-bd04-08aed798c952.png)

![image](https://user-images.githubusercontent.com/1894533/207062842-10ba90f6-67f3-4f7b-9fe4-0cc6f5747cfd.png)

Reply:
Thanks for mentioning it. I fixed and included the fix in this PR.